api/swagger.yml

```diff
@@ -1274,6 +1274,39 @@ components:
       required:
         - pattern
 
+    ImportPath:
+      type: object
+      required:
+        - path
+        - destination
+        - type
+      properties:
+        path:
+          type: string
+          description: A source location to ingested path or to a single object. Must match the lakeFS installation blockstore type.
+          example: s3://my-bucket/production/collections/
+        destination:
+          type: string
+          description: Destination for the imported objects on the branch 
+          example: collections/
+        type:
+          type: string
+          enum: [ prefix, object ]
+          description: Path type, can either be 'prefix' or 'object'
+
+    ImportCreation:
```

Comment:
We should probably mark the existing APIs this replaces as deprecated, I assume there's a Swagger-y way to do so

Reply:
Not sure we want to deprecate them, I think this requires a discussion as per the PR description states.